### PR TITLE
Implement workaround for jQuery event normalization

### PR DESCRIPTION
**Pull request by breeze4**
See discussions in issues #311 and #298.

I implemented the suggested fix by @sudiptaangular.

Without this fix, you can't use jQuery with Angular and instead have to fall back to jqLite. jQuery normalizes the event object to match the W3C spec, so any added fields get removed. There is the "originalEvent" object that can work for this.

Legal: I give full permission to use this code in any way.

### CODE REVIEW

**Comment by uncedric** [comment]
Why the test failed? It would be amazing to have a feature like this merged.

**Comment by breeze4** [comment]
@uncedric , my understanding from looking at the travis build is that it's timing out due to something with selenium perhaps? I saw that @mcasimir has another branch "pipeline-refactor" (see https://travis-ci.org/mcasimir/mobile-angular-ui/builds/119080983 ) that he's been working on and fixing tests.

With that in mind, I created another version of the patch with pipeline-refactor changes merged and the tests all pass fine now, so I don't think I broke anything with my changes. 

Bottom line: anyway that @mcasimir would prefer, I can make a PR for that merged branch or this can be merged in.

**Comment by mcasimir** [comment]
@breeze4 Great PR! thanks!
@breeze4 @uncedric

I'm working on pipeline right now.. before i was using browserstack and i gave a try to soucelabs but both disappointed me for different reason. I found soucelabs appealing but the integration with karma is extremely unreliable (tests passing without being executed). 

Currently i'm playing with docker and selenium to provide a more robust test phase.

About merging this PR, my thought is:
- if i manage to merge `pipeline-refactor` branch soon enough i'd like to merge the PR afterwards
- if it takes too long (ie. travis complaining about some docker networking stuffs) i think we can ignore failing tests and after a code review we can merge directly to master and then i'll rebase `pipeline-refactor`.

Anyway i'll get back to you in a couple of days.

Thanks!!

**Comment by mcasimir** [comment]
@breeze4 it took ages, but i've merged the branch and of course it complains about outdated code. Let me know if u have the time to rebase and re-push, otherwise i'll figure out a way.
